### PR TITLE
Add replaceNSEName chain element

### PR DESCRIPTION
**Pull request by glazychev-art**
Signed-off-by: Artem Glazychev <artem.glazychev@xored.com>

## Description
Split `replacelabels` chain element to `replacelabels` and `replaceNSEName`
We need to use already discovered NSE on refresh

## Issue link
https://github.com/networkservicemesh/deployments-k8s/issues/6305

## How Has This Been Tested?

- [x] Added unit testing to cover
- [ ] Tested manually
- [x] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [x] Bug fix
- [x] New functionallity
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: pkg/networkservice/common/replacelabels/client.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Doc.ai and/or its affiliates.
```
Comment (denis-tingaikin) [inline pkg/networkservice/common/replacelabels/client.go:1]: Should we return this?

File: pkg/networkservice/chains/nsmgr/suite_test.go [hunk 2]
```diff
@@ -795,6 +796,7 @@ func additionalFunctionalityChain(ctx context.Context, clientURL *url.URL, clien
 					client.WithAdditionalFunctionality(
 						mechanismtranslation.NewClient(),
 						replacelabels.NewClient(labels),
+						replacensename.NewClient(),
```
Comment (denis-tingaikin) [inline pkg/networkservice/chains/nsmgr/suite_test.go:799]: Im mostly sure that we could face with the similar issue in future (when we need to replace something in pass through requests)

Should we create a pkg like `passthrough` and concatenate all needed  replace elements?

Something like

```go
package passthrough

func NewClient(labels map[string]string) {
return chain.NewClient(
replacelabels.NewClient(labels),
replacensename.NewClient(),
)

}
``` 

And use `passthrough` chain in firewall nses?


/cc @edwarnicke

Comment (glazychev-art) [reply pkg/networkservice/chains/nsmgr/suite_test.go]: It's a good idea, thanks! 
Done

File: pkg/networkservice/common/replacelabels/client.go (hunk 1 above)
Comment (glazychev-art) [reply pkg/networkservice/common/replacelabels/client.go:1]: Yes, done

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**